js/kubewarden/host_capabilities/oci/manifest_config/manifest_config.test.ts

```diff
@@ -0,0 +1,154 @@
+import * as HostCall from '../../index';
+import { MediaTypeImageManifest, MediaTypeImageConfig } from '../oci-spec';
+
+import { ManifestConfig } from './manifest_config';
+import type { OciImageManifestAndConfigResponse } from './types';
+
+class MockWasmHost {
+  private mockResponse: any;
+  private knownImages: Record<string, any>;
+
+  constructor(knownImages: Record<string, any> = {}) {
+    this.knownImages = knownImages;
+    this.mockResponse = null;
+  }
+
+  hostCall(
+    binding: string,
+    namespace: string,
+    operation: string,
+    payload: ArrayBuffer,
+  ): Uint8Array {
+    if (binding !== 'kubewarden' || namespace !== 'oci' || operation !== 'v1/oci_manifest_config') {
+      throw new Error('Invalid host call parameters');
+    }
+
+    const image = JSON.parse(new TextDecoder().decode(payload)) as string;
+
+    if (this.mockResponse) {
+      return new TextEncoder().encode(JSON.stringify(this.mockResponse));
+    }
+
+    if (this.knownImages[image]) {
+      return new TextEncoder().encode(JSON.stringify(this.knownImages[image]));
+    }
+
+    throw new Error(`Image not found: ${image}`);
```

Comment:
```suggestion
    throw new Error(`image not found: ${image}`);
```


On the e2e tests we have `image` lowercase, we should be consistent.